eng/tools/lint-diff/src/markdown-utils.ts

```diff
@@ -164,60 +92,34 @@ export async function getRelatedArmRpcFromDoc(ruleName: string): Promise<string[
     return rpcRules;
   }
 
-  let walker = new commonmark.Parser().parse(response.data).walker();
-  let event;
-  while ((event = walker.next())) {
-    const node = event.node;
+  // Use marked to parse the markdown and extract the related ARM guideline codes
```

Comment:
For my understanding, why does LintDiff itself need to parse markdown?

Reply:
This code extracts the ARM RPC codes from docs. For example: https://github.com/Azure/azure-openapi-validator/blob/main/docs/consistent-response-schema-for-put.md

![image](https://github.com/user-attachments/assets/3092081e-9cd6-4896-9efb-99b81bcd0715)

This is used in the output table.

The code for this is specific to LintDiff but it could be applicable anywhere that results for openapi validator are interpreted.